### PR TITLE
Reexport/tokio serde

### PR DESCRIPTION
**Pull request by Frederik-Baetens**

### CODE REVIEW

File: example-service/src/server.rs [hunk 1]
```diff
@@ -11,11 +11,11 @@ use std::{
     io,
     net::{IpAddr, SocketAddr},
 };
+use tarpc::tokio_serde::formats::Json;
```
Comment (tikue) [inline example-service/src/server.rs:14]: nit: put this in the `use tarpc::{ ... }` block below.

File: example-service/src/client.rs [hunk 2]
```diff
@@ -6,8 +6,8 @@
 
 use clap::{App, Arg};
 use std::{io, net::SocketAddr};
+use tarpc::tokio_serde::formats::Json;
```
Comment (tikue) [inline example-service/src/client.rs:9]: nit: put this in the `use tarpc::{ .. }` block below.

**Review by tikue — CHANGES_REQUESTED** [review]
Two very minor nits, otherwise LGTM :)

File: example-service/src/client.rs (hunk 2 above)
Comment (Frederik-Baetens) [reply example-service/src/client.rs]: Both done :)

**Comment by tikue** [comment]
Thank you!

**Pull request merged**